sanssouci/post_hoc_bounds.py

```diff
@@ -93,6 +93,35 @@ def min_tp(p_values, thr):
     return p_values.shape[0] - max_fp(p_values, thr)
 
 
+def min_tdp(p_values, thr):
+    """
+    Lower bound for the proportion of true discoveries in a selection
+
+    Parameters
+    -----------
+
+    p_values : 1D numpy.array
+        A 1D numpy array of p-values for the selected items
+    thr : 1D numpy.array
+        A 1D numpy array of non-decreasing k-FWER-controlling thresholds
+
+    Returns
+    -------
+
+    ndarray or scalar :
```

Comment:
Is it possible to decide once and for all whether it's a fload or an array ? Changing signatures are a source of complexity. If you really can't avoid this, then explain in which casse it shall be a float or an array.